edx_sga/sga.py

```diff
@@ -680,13 +704,13 @@ def upload_allowed(self):
         """
         return not self.past_due() and self.score is None
 
-    def _file_storage_path(self, sha1, filename):
+    def file_storage_path(self, sha1, filename):
         # pylint: disable=no-member
         """
         Get file path of storage.
         """
         path = (
-            '{loc.org}/{loc.course}/{loc.block_type}/{loc.block_id}'
+            u'{loc.org}/{loc.course}/{loc.block_type}/{loc.block_id}'
```

Comment:
Why do we need this unicode marker here? If you want to return a unicode string, please use `six.u` instead